src/language/providers/completionProvider.ts

```diff
@@ -543,6 +544,12 @@ export const completionProvider = languages.registerCompletionItemProvider(
 );
 
 const getDefaultSchema = (): string => {
+  const instance = getInstance();
+  const config = instance.getConfig();
+
   const currentJob = JobManager.getSelection();
-  return currentJob && currentJob.job.options.libraries[0] ? currentJob.job.options.libraries[0] : `QGPL`;
+  const currentSchema = currentJob ? currentJob.job.options.libraries[0] : undefined;
+  
+  const schema = currentJob && currentSchema !== `*LIBL` && currentSchema !== undefined ? currentJob.job.options.libraries[0] : (config.currentLibrary || `QGPL`);
```

Comment:
Nitpicking here, eliminate the constant and just return the value.

Reply:
Good point. I think I put it there to debug.